public/js/main.js

```diff
@@ -1,31 +1,20 @@
-/* global window, document, DebuggerConfig */
-
+const React = require("react");
 const { bindActionCreators, combineReducers } = require("redux");
-const { Provider } = require("react-redux");
 const ReactDOM = require("react-dom");
-const React = require("react");
-const { DOM: dom } = React;
 
-const DevToolsUtils = require("devtools-sham/shared/DevToolsUtils");
-const AppConstants = require("devtools-sham/sham/appconstants").AppConstants;
-const { injectGlobals } = require("./utils/debug");
-const { isEnabled, isFirefoxPanel, getValue,
-        isDevelopment, setConfig } = require("../../config/feature");
+const {
+  client: { getClient, firefox },
+  renderRoot,
+} = require("devtools-local-toolbox");
 
-setConfig(DebuggerConfig);
-
-// Set various flags before requiring app code.
-if (isEnabled("logging.client")) {
-  DevToolsUtils.dumpn.wantLogging = true;
-}
+const { getValue, isDevelopment, isFirefoxPanel } = require("Feature");
 
-const { getClient, connectClients, startDebugging } = require("./clients");
-const firefox = require("./clients/firefox");
+const { bootstrap } = require("devtools-local-toolbox");
 const configureStore = require("./utils/create-store");
+
 const reducers = require("./reducers");
 const selectors = require("./selectors");
 
-const LandingPage = require("./components/LandingPage");
 const App = require("./components/App");
 
 const createStore = configureStore({
```

Comment:
I see that we still are creating our own store, so that's good. I thought that this code had been copied over too.